### ISSUE TITLE
Arlo Battery Levels

### ISSUE THREAD

**Issue by giffordj** [twrecked/hass-aarlo#363]
Battery level sensors are returns unknown value.

cards:
  - entities:
      - entity: sensor.aarlo_battery_level_back_left
    title: Battery Status
    type: entities

**Comment by twrecked** [comment]
Did you give admin access to the HA account?

**Comment by giffordj** [comment]
Yes

**Comment by twrecked** [comment]
Can you turn on [debug](https://github.com/twrecked/hass-aarlo#debugging) and look for `batteryLevel` in the output. It should look something like this:

```
2021-01-02 23:08:24 DEBUG (ArloEventStream) [custom_components.aarlo.sensor] callback:Battery Level Basement Camera:batteryLevel:59
```

**Comment by giffordj** [comment]
The only thing I get is batterylevel sensor created.

Here's what I have in my logger.

  custom_components.aarlo: debug
  custom_components.aarlo.alarm_control_panel: debug
  custom_components.aarlo.binary_sensor: debug
  custom_components.aarlo.camera: debug
  custom_components.aarlo.light: debug
  custom_components.aarlo.media_player: debug
  custom_components.aarlo.sensor: debug
  custom_components.aarlo.switch: debug
  pyaarlo: debug

**Comment by twrecked** [comment]
Do you have 2FA enabled on this account?

**Comment by giffordj** [comment]
No, I don't.

**Comment by twrecked** [comment]
Did you ever get this working?

What type of cameras do you use?

**Comment by giffordj** [comment]
No, I didn't.  It's just the standard arlo camera, mix of first and second
gen.

**Comment by twrecked** [comment]
That's all the cameras I have. Can you try using your Home Assistant account on my.arlo.com and see if they appear there.

**Comment by giffordj** [comment]
They do.

**Comment by twrecked** [comment]
I'm kinda stumped on this one.

If you want to point one of the cameras at something insignificant and share it with me - steve.herrell@gmail.com - I can take a look here.
